### PR TITLE
chore: improve dev workflow

### PR DESCRIPTION
**Pull request by satya164**
Includes #101 and #102.

### CODE REVIEW

File: example/rn-cli.config.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-commonjs */
```
Comment (ahmedlhanafy) [inline example/rn-cli.config.js]: @satya164 is this for watching changes in the library instead of manually copying?

File: package.json [hunk 2]
```diff
@@ -3,44 +3,48 @@
...
+    "react-native": "~0.43.2",
+    "react-native-vector-icons": "~4.1.1"
   },
   "dependencies": {
     "color": "^0.11.4",
-    "highlight.js": "^9.8.0",
+    "prop-types": "^15.5.10",
     "react-native-drawer": "^2.3.0"
+  },
+  "scripts": {
```
Comment (ahmedlhanafy) [inline package.json]: @satya164 Just curious why did you move scripts to the bottom?

File: example/rn-cli.config.js (hunk 1 above)
Comment (satya164) [inline example/rn-cli.config.js]: yes

File: package.json (hunk 2 above)
Comment (satya164) [inline package.json]: just easier to find at bottom

File: example/rn-cli.config.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      'prop-types',
+      'react-native-drawer',
+      'color',
+    ];
+  },
+  getBlacklistRE() {
+    /* eslint-disable prettier/prettier */
+    return blacklist([
+      new RegExp(
+        `^${escape(path.resolve(__dirname, '..', 'node_modules'))}\\/.*$`,
```
Comment (ahmedlhanafy) [inline example/rn-cli.config.js]: The trailing comma crashes the packager for me

File: example/rn-cli.config.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      ),
+      new RegExp(
+        `^${escape(path.resolve(__dirname, '..', 'docs', 'node_modules'))}\\/.*$`,
```
Comment (ahmedlhanafy) [inline example/rn-cli.config.js]: Same here

**Comment by ahmedlhanafy** [comment]
@satya164 Other than the trailing comma issue in `rn-cli.config.js`, everything is working fine 👍

**Comment by satya164** [comment]
Damn, the `eslint-disable` didn't work for some reason. Fixed.
